data-raw/qc-markdowns/rst/lower-sac/knights_landing/knights_camp_data_pull.Rmd

```diff
@@ -0,0 +1,391 @@
+---
```

Comment:
A lot of the tables generated in the Rmd are super long. The HTML does not seem that useful to look at. If we want it to be a more useful documentation resource we should update the display formating. 

I think fine for now but something we could update in the furture

Reply:
same comment for tisdale & feather

Reply:
I agree with you. i made some minor edits that at least make the html easier to read. the goal is to see the output tables pushed to google cloud. for future we can consider ways to make this better.